EndlessClient/Dialogs/GuildDialog.cs

```diff
@@ -229,5 +287,20 @@ private void SetStateIfGuildMember(GuildDialogState state)
 
             SetState(state);
         }
+
+        private void ShowChangeDescriptionMessageBox()
+        {
+            // TODO: Move max description length to config?
```

Comment:
If this is a hard-coded constant in the vanilla client, I'd say make it a hard-coded constant in EndlessClient as well. No need for config on this.

Reply:
Removed comment